tests/performance/pom.xml

```diff
@@ -150,6 +151,76 @@
             </build>
         </profile>
         <profile>
+            <id>micro-benchmark</id>
+            <build>
+                <plugins>
+                    <plugin>
```

Comment:
Why is this needed? Are we running JMH tests and also running benchmark tests cases with same profile?